### DISCUSSION TITLE
Olimex ESP32-POE -ISO bootet bei Aufruf der Web UI, wenn Rechner und ESP am gleichen Switch hängen

### DISCUSSION THREAD

**Discussion by flattermues** [tbnobody/OpenDTU#1816] [Q&A] [answered]
Ich habe ein Problem, was ich mir nicht erklären kann, was ich aber auch nicht als BUG einstufen will.
Firmwareversion ist v24.2.16, der Olimex ist im Moment noch ohne Funkmodul, es ist aber bereits ein CMT2300A konfiguriert.
Wenn das Teil in meinem WLAN ist, kann ich die Web UI ganz normal über den Browser meines Desktop erreichen (Firefox 123.0 auf LMDE).
Sobald ich es über den gleichen Switch, an dem auch mein Desktop steckt, anschließe und ich die Web UI aufrufe, bootet es. Ich habe das Ganze über ein Terminal mitgeschrieben [capture.txt](https://github.com/tbnobody/OpenDTU/files/14526533/capture.txt) und man sieht, daß das Umschalten auf LAN noch funktioniert (Zeilen 40-48), aber sobald ich im Browser die Seite aufrufe, kommt es zur Katastrophe (Zeile 49).
Rufe ich die Web UI mit gestecktem LAN von einem beliebigen anderen Rechner auf, funktioniert es problemlos.
Was es jetzt völlig absurd erscheinen läßt: Als ich das Teil gestern mit dem Web Flasher frisch geflashed und konfiguriert hatte, funktionierte es noch, sowohl mit WLAN als mit LAN, dann über Nacht ausgeschaltet und heute nun dieses mir unerklärliche Verhalten.

**Comment by vaterlangen** [comment, marked as answer]
Deine Beschreibung klingt mir sehr nach #1705

**Reply by flattermues** [reply]
Vielen Dank für den Hinweis. Es läuft zwar in der jetzigen Konfiguration problemlos, aber ich werde ihm bei Gelegenheit ein Update verpassen.
